attendees/deltabug.txt

```diff
@@ -0,0 +1,6 @@
+Hello,
+
+I am Deltabug. This is my first PyCon.
+I like Dragons
```

Comment:
You should spell this with a lowercase "d" :wink:

Reply:
Hehe yes! In Twitter it's a capital "D" and I forget 😁